Hudson Build Failing due to network issue

It looks like the changes made in https://github.com/eclipse/smarthome/pull/829 broke the nightly build, https://hudson.eclipse.org/smarthome/job/SmartHomeDistribution-Nightly/749/console

Reading the error, `[ERROR] Failed to execute goal com.github.eirslett:frontend-maven-plugin:0.0.27:install-node-and-npm (Install node and npm) on project org.eclipse.smarthome.ui.paper: Could not download Node.js: Could not download https://nodejs.org/dist/v5.4.0/node-v5.4.0-linux-x64.tar.gz: Network is unreachable -> [Help 1]`. It looks like it was unable to reach the internet and download the requested nodeJS .gz. 

Is there something in Hudson that is blocking network calls?

--- a/extensions/ui/org.eclipse.smarthome.ui.paper/pom.xml
+++ b/extensions/ui/org.eclipse.smarthome.ui.paper/pom.xml
@@ -39,6 +39,8 @@
                         <configuration>
                             <nodeVersion>v5.4.0</nodeVersion>
                             <npmVersion>3.3.12</npmVersion>
+                            <nodeDownloadRoot>http://nodejs.org/dist/</nodeDownloadRoot>
+                            <npmDownloadRoot>http://registry.npmjs.org/npm/-/</npmDownloadRoot>
                             <!-- optional: where to download node and npm from. Defaults to http://nodejs.org/dist/ -->
                             <!--<downloadRoot>https://nodejs.org:80/dist/</downloadRoot>-->
                         </configuration>

